src/ua/dp/ort/OOP_10/ForeignPassport.java

```diff
@@ -0,0 +1,71 @@
+package ua.dp.ort.OOP_10;
+
+/**
+ * This class is a foreign passport of Ukraine
+ * and it is an extension class Passport
+ * Class contains fields which are other classes  
+ * @author Admin
+ *
+ */
+public class ForeignPassport extends Passport {
+	private Visa _visaData;
+	private ForeignPassportForei _foreignPassportForei;
+		
+	/**
+	 * constructor without parameters

```

Comment:
Do not write obvious things in javadocs. This comment does not help me at all. What default values are set for fields?